frontend/src/utils/constants.js

```diff
@@ -114,7 +114,7 @@ export const APPLICATION_ERROR_MESSAGES = Object.freeze({
   EXPENSE_AUTHORITY: 'Expense authority required',
   LICENCE_INFO: 'Licence information required',
   LICENCE_CONFIRMATION: 'Confirmation of licence information required',
-  DOCUMENT_UPLOAD: 'Document upload required',
```

Comment:
we may need an error message for Rent/Lease document required.

Reply:
You are correct, i've added.